### PR TITLE
New package: nkdAgility.AzureDevOpsExportActionableAgile version 0.1.1

### PR DESCRIPTION
**Pull request by MrHinsh**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140904)

### CODE REVIEW

**Comment by MrHinsh** [comment]
I can confirm that this installes locally.

**Comment by stephengillie** [comment]
Hi @MrHinsh,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 821.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335187 2024-02-25 06:33:50.440 [FAIL] Installer failed security check. Url: https://github.com/nkdAgility/azure-devops-export-for-actionable-agile/releases/download/v0.1.1/AzureDevOps.Export.ActionableAgile-0.1.1.zip Result: 0x80004005 2024-02-25 06:33:50.440 [CLI ] Terminating context: 0x8a15002d at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DownloadFlow.cpp:1e6

(Automated response - build 821.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\nkdAgility.AzureDevOpsExportActionableAgile__DefaultSource\AzureDevOps.Export.ActionableAgile.ConsoleUI.exe returned exit code: -2147450749
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\AzureDevOpsExport.exe returned exit code: -2147450726

(Automated response - build 860.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Links\AzureDevOpsExport.exe The application to execute does not exist: 'C:\Users\User\AppData\Local\Microsoft\WinGet\Links\AzureDevOps.Export.ActionableAgile.ConsoleUI.dll'. PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\nkdAgility.AzureDevOpsExportActionableAgile__DefaultSource\AzureDevOps.Export.ActionableAgile.ConsoleUI.exe You must install .NET to run this application.  App: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\nkdAgility.AzureDevOpsExportActionableAgile__DefaultSource\AzureDevOps.Export.ActionableAgile.ConsoleUI.exe Architecture: x64 App host version: 8.0.2 .NET location: Not found  Learn more: https://aka.ms/dotnet/app-launch-failed  Download the .NET runtime: https://aka.ms/dotnet-core-applaunch?missing_runtime=true&arch=x64&rid=win-x64&os=win10&apphost_version=8.0.2

(Automated response - build 869.)

**Comment by stephengillie** [comment]
The package installs and launches normally, but the symlink still gets confused:

> PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Links\AzureDevOpsExport.exe The application to execute does not exist: 'C:\Users\User\AppData\Local\Microsoft\WinGet\Links\AzureDevOps.Export.ActionableAgile.ConsoleUI.dll'. 

image

File: manifests/n/nkdAgility/AzureDevOpsExportActionableAgile/0.1.1/nkdAgility.AzureDevOpsExportActionableAgile.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: nkdAgility.AzureDevOpsExportActionableAgile
+PackageVersion: 0.1.1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: AzureDevOps.Export.ActionableAgile.ConsoleUI.exe
+  PortableCommandAlias: AzureDevOpsExport
+Installers:
```
Comment (stephengillie) [inline manifests/n/nkdAgility/AzureDevOpsExportActionableAgile/0.1.1/nkdAgility.AzureDevOpsExportActionableAgile.installer.yaml:11]:
```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.DotNet.Runtime.8
Installers:
```

(Automated response - build 869.)

Comment (MrHinsh) [reply manifests/n/nkdAgility/AzureDevOpsExportActionableAgile/0.1.1/nkdAgility.AzureDevOpsExportActionableAgile.installer.yaml:11]: Is a dependancy the right thing to add when its a portable package?

**Comment by stephengillie** [comment]
> With the dependency, the package installs and launches normally, but the symlink still gets confused:
> 
> > PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Links\AzureDevOpsExport.exe The application to execute does not exist: 'C:\Users\User\AppData\Local\Microsoft\WinGet\Links\AzureDevOps.Export.ActionableAgile.ConsoleUI.dll'.
> 
> image

This is unchanged.

**Comment by MrHinsh** [comment]
Yes, portable CLI tools do not work if installed via admin.

**Comment by MrHinsh** [comment]
This is blocked by #2711

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Trenly** [comment]
@MrHinsh - Support for zipped binaries has now been added and is available in the pipelines. This will require you to update your submission to ManifestVersion: 1.9.0 and set ArchiveBinariesDependOnPath: true in the installer node. To keep things clean, this PR is being closed. Please submit a new PR for the latest version of this package. Thank you for your contributions!

**Pull request closed without merging**